### PR TITLE
Add option to specify more than one rabbit queue

### PR DESCRIPTION
**Pull request by melindaloubser1**
- Add template to include list of multiple rabbit queues as is possible in a docker-compose deployment for adding e.g. a queue for ELK to consume from

### CODE REVIEW

File: charts/rasa-x/templates/rasa-deployments.yaml [hunk 1]
```diff
@@ -127,8 +127,6 @@ spec:
               key: {{ template "rasa-x.redis.password.key" $ }}
         {{- end }}
         {{- if $.Values.rabbitmq.enabled }}
-        - name: "RABBITMQ_QUEUE"
-          value: "{{ $.Values.rasa.rabbitQueue }}"
```
Comment (melindaloubser1) [inline charts/rasa-x/templates/rasa-deployments.yaml:131]: This was a duplicate I added accidentally 4 months ago

**Review by desmarchris — APPROVED** [review]

**Review by tczekajlo — APPROVED** [review]
🎖️

File: charts/rasa-x/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 
-version: "1.11.0"
+version: "1.11.1"
```
Comment (tczekajlo) [inline charts/rasa-x/Chart.yaml:4]:
```suggestion
version: "1.11.0"
```
We add a new feature, not a fix :D

**Review by tczekajlo — CHANGES_REQUESTED** [review]

**Review by tczekajlo — APPROVED** [review]
🎖️

**Pull request merged**